services/webnn/ort/ort_model_editor.h

```diff
@@ -29,6 +31,8 @@ constexpr int32_t kOrtOpsetVersion = 21;
 // EPContext op is used for exporting the EP context cache model.
 // https://onnxruntime.ai/docs/execution-providers/EP-Context-Design.html#onnxruntime-ep-context-cache-feature-design
 constexpr int32_t kEPContextOpsetVersion = 1;
+constexpr int32_t kMSNchwcDomainOpsetVersion = 1;
+constexpr int32_t kMSInternalNHWCDomainOpsetVersion = 1;
```

Comment:
Could you please educate me how to set the opset version? Any references?

Reply:
I find `This version of the operator has been available since version 1 of the 'com.microsoft' operator set.`  at https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md.

But for the two 
`constexpr char kMSNchwcDomainName[] = "com.microsoft.nchwc";`
`constexpr char kMSInternalNHWCDomainName[] = "com.microsoft.internal.nhwc";`,
I can only find in the ort repo from `onnxruntime\contrib_ops\cpu\cpu_contrib_kernels.cc` as below and select the lowest `1`:
![image](https://github.com/user-attachments/assets/06648f2b-63c2-48c9-92a4-85cfd0af1ff0)

![image](https://github.com/user-attachments/assets/c6179471-a162-42c9-a9cf-72334951c104)